<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deathScreen</class>
 <widget class="QDialog" name="deathScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="deathImageLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replayButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>540</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>REPLAY</string>
   </property>
  </widget>
  <widget class="QLabel" name="globalHighScore">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GLOBAL HIGH SCORE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yourScore">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>YOUR SCORE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yourHighScore">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>YOUR HIGH SCORE:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
